lib/manageiq/floe/workflow/path.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module ManageIQ
+  module Floe
+    class Workflow
+      class Path
+        def initialize(payload, context)
+          @payload = payload
+          @context = context
+        end
+
+        def value(input = {})
+          obj, path =
+            if payload.start_with?("$$")
+              [context, payload[1..]]
+            else
+              [input, payload]
+            end
+
+          results = JsonPath.on(obj, path)
+
+          results.count < 2 ? results.first : results
```

Comment:
NOTE I'm not sure if there is a better way to handle this, but `JsonPath` returns an array even if a single element was matched.  This means we can't do a simple 
```
{"foo" => "bar"}
"$.foo" -> "bar"
```
it will end up as `"$.foo -> ["bar"]`